packages/provider_approach/lib/features/profile/preview/profile_widget.dart

```diff
@@ -186,6 +178,28 @@ class ProfileWidget extends StatelessWidget {
     );
   }
 
+  AppBar _buildAppBar(BuildContext context, ProfileModel model) {
```

Comment:
Is `BuildContext context` argument needed? Looks like it never using inside method.